web/client/plugins/MapSearch.jsx

```diff
@@ -14,12 +14,22 @@ const ConfigUtils = require('../utils/ConfigUtils');
 * @name MapSearch
 * @memberof plugins
 * @class
+* @prop {boolean} splitTools used to to separate search and remove buttons in toolbar
```

Comment:
to to <-- typo.
please explain the purpose of this flag, not only the generic usage context